webpack/JobWizard/__tests__/fixtures.js

```diff
@@ -48,6 +48,30 @@ export const jobTemplateResponse = {
       default: '',
       hidden_value: false,
     },
+    {
+      name: 'adv resource select',
+      required: false,
+      input_type: 'user',
+      options: [
+        {
+          name: 'resource1',
+          id: 43,
+        },
+        {
+          name: 'resource2',
+          id: 10,
+        },
+        {
+          name: '',
+          id: 11,
+        },
+      ],
+      advanced: true,
+      value_type: 'plain',
```

Comment:
do we need to also add a fixture of [value_type: 'resource](https://github.com/theforeman/foreman_remote_execution/pull/644/files#diff-494fb439c4dcbef79b8470f33ba943aa649b2d8e698a9bd26c5353e9e33158f7R28)' ?